Makefile

```diff
@@ -20,6 +20,13 @@ endif
 PROGRAM ?= hello
 TARGET ?= sifive-hifive1
 
+# Coremark require PORT_DIR set for different OS, freedom-metal for us!
+ifeq ($(PORT_DIR),)
+ifeq ($(PROGRAM),coremark)
```

Comment:
Per comment, PORT_DIR is a coremark thing, so I think it make more sense to flip the check.

Reply:
patch updated.